src/plugins/bodhi.c

```diff
@@ -271,6 +270,36 @@ static void bodhi_print_errors_from_json(json_object *json)
     return;
 }
 
+/**
+ * Parses only name of rpm package
+ */
+static int parse_pkginfo_name(const char *pkg_full_name, char **name)
```

Comment:
Oh dear, you are right. I've missed the `c <= pkg_full_name` test. Sorry! Please, ignore my initial comment.

However, I am wondering what format of package name the function should parse. Is it ```foo-1.2.3-4.fc26.x86_64``` or without the arch part?